lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -775,27 +775,54 @@ verify_check => md5_hex($check->test_query),
     $sth->finish();
 
     my $heading = { map { $_ => $_ } @{$check->display_cols} };
-    my $buttons = [
+    my %buttons = map { $_ => 1 } @{$check->buttons};
+    my $buttons;
+    push @$buttons,
            { type => 'submit',
              name => 'action',
-            value => $check->columns ? 'fix_tests' : 'cancel',
-             text => $request->{_locale}->text($check->columns
-                                                ? 'Save and Retry'
-                                                : 'Cancel'),
+            value => 'fix_tests',
+          tooltip => { id => 'action-fix-tests',
+                       msg => $request->{_locale}->text($check->{tooltips}{'Save and Retry'}),
+                       position => 'above'
+                     },
+             text => $request->{_locale}->text('Save and Retry'),
             class => 'submit' },
-    ];
+        if $check->columns;
+    push @$buttons,
+           { type => 'submit',
+             name => 'action',
+            value => 'cancel',
+          tooltip => { id => 'action-cancel',
+                       msg => $request->{_locale}->text($check->{tooltips}{'Cancel'}),
+                       position => 'above'
+                     },
+             text => $request->{_locale}->text('Cancel'),
+            class => 'submit' }
+    if $buttons{Cancel} or scalar($check->columns) == 0;
+    push @$buttons,
+           { type => 'submit',
+             name => 'action',
+            value => 'force',
+          tooltip => { id => 'action-force',
+                       msg => $request->{_locale}->text($check->{tooltips}{'Force'}),
+                       position => 'above'
+                     },
+             text => $request->{_locale}->text('Force'),
+            class => 'submit' }
+    if $buttons{Force} && $check->{force_queries};
 
     my $template = LedgerSMB::Template->new(
         path => 'UI',
         template => 'form-dynatable',
-        format => 'HTML',
+        format => 'HTML'
     );
 
     return $template->render_to_psgi({
            form               => $request,
            base_form          => 'dijit/form/Form',
            heading            => $heading,
-           headers            => [$check->display_name, $check->instructions],
+           headers            => [$request->{_locale}->text($check->display_name),
```

Comment:
any occurrance of `text(<string>)` means the string needs to be translated. The scanner will find here that the string can't be translated due to the fact that it's not a string constant.

Instead of using `text(<variable>, <@parameters>)`, please use `maketext(<variable>, <@parameters>)`; the latter doesn't get marked for translation, but does do the translation.